test/cross-browser/reference.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <!-- meta -->
+        <meta charset="utf-8" />
+        <title>rc-socket reference implementation</title>
+
+        <!-- external script -->
+        <script src="../../dist/rc-socket.js"></script>
+
+        <!-- styles -->
+        <!--<link rel="stylesheet" href="../node_modules/easy-build/node_modules/mocha/mocha.css" />-->
+    </head>
+
+    <!--<script src="http://cdnjs.cloudflare.com/ajax/libs/loglevel/1.4.0/loglevel.min.js"></script>-->
+
+    <!-- content -->
+    <div id="workboard" style="position: absolute;"></div>
+    <div id="mocha">
+        <input type="button" onclick="createRcSocket();" value="StartSocket"/>
+    </div>
+
+    <!-- traces -->
+    <script>
+        var childWin;
+
+        var createRcSocket = function() {
+            var child = window.open('', 'childLogger');
+            childWin = child.window;
+
+            if (!childWin.clientLogger) {
+                childWin.clientLogger = {
+                    buffer: new Array(),
```

Comment:
In js land, it is common to use literals `[]`.

While I am suggesting this purely for style/consistency, it actually performs better as well: http://stackoverflow.com/questions/7375120/why-is-arr-faster-than-arr-new-array